### PR TITLE
fix lint warnings in migrationExample

### PR DESCRIPTION
**Pull request by zaki50**
@realm/java

### CODE REVIEW

File: examples/migrationExample/lint.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<lint>
+    <issue id="AllowBackup" severity="ignore" />
+    <issue id="IconLauncherShape" severity="ignore" />
+    <issue id="IconMissingDensityFolder" severity="ignore" />
+    <issue id="GoogleAppIndexingWarning" severity="ignore" />
+    <issue id="PrngFix" severity="ignore" />
```
Comment (nhachicha) [inline examples/migrationExample/lint.xml]: remove please

**Comment by cmelchior** [comment]
:+1:

**Comment by nhachicha** [comment]
:+1:

**Comment by dalinaum** [comment]
:+1:

**Comment by stk1m1** [comment]
👍

**Comment by beeender** [comment]
:+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1::+1:

**Comment by zaki50** [comment]
thanks everyone especially @beeender!!

**Comment by cmelchior** [comment]
I am not sure there is enough :+1: in this PR :grin:

**Pull request merged**